src/CaptureFile/include/CaptureFile/CaptrureSectionInputStream.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
The filename has a typo (captrure -> capture).

Reply:
done.